docs/environment-variables.md

```diff
@@ -63,25 +63,38 @@ The OAuth token secret, generated by Launchpad.
 ## GitHub settings
 
 ### GITHUB_API_ENDPOINT
-- **Example**: `http://localhost:4000/github`, `https://api.github.com`
+- **Example**: `http://localhost:4000/github`
+- **Example**: `https://api.github.com`
 
 The base URL for the GitHub API.
 
-### GITHUB_USERNAME
-- **Example:** katiefenn
-- **DEPRECATED**
+### GITHUB_AUTH_LOGIN_URL
+- **Example**: `http://localhost:4000/github/login/oauth/authorize`
+- **Example**: `https://github.com/login/oauth/authorize`
 
-The GitHub account name to use for authenticating with the GitHub API.
-**WARNING: Do not commit GitHub credentials to the repository. Keep them secure on your computer.**
-**WARNING: This setting is deprecated and subject to removal when Oauth authentication is implemented**
+The page users should be redirected to to begin logging in.
 
-### GITHUB_PASSWORD
-- **Example:** password
-- **DEPRECATED**
+### GITHUB_AUTH_VERIFY_URL
+- **Example**: `http://localhost:4000/github/login/oauth/access_token`
+- **Example**: `https://github.com/login/oauth/access_token`
 
-The GitHub account password to use for authenticating with the GitHub API.
-**WARNING: Do not commit GitHub credentials to the repository. Keep them secure on your computer.**
-**WARNING: This setting is deprecated and subject to removal when Oauth authentication is implemented**
+The GitHub API endpoint for exchanging Oauth tokens.
+
+### GITHUB_AUTH_REDIRECT_URL
+-- **Example**: `http://localhost:8000/login/verify`
+-- **Example**: `https://build.snapcraft.io/login/verify`
+
+The local page users should be redirected to after logging in with GitHub.
+
+### GITHUB_AUTH_CLIENT_ID
+-- **Example:** `389a7b4c2ade662c0bf4`
+
+The GitHub application id. Created using the form at GitHub > Your profile > Oauth Applications > Register a new application. Shared credentials for dev, staging and production sites are stored on LastPass. Access to credentials can be obtained by asking a developer.
+
```

Comment:
Let's not share accounts; we should have a specific account just for this service.  (Possibly one for staging and one for production, though that may be overkill since they're both devops.)